src/screens/pos-actions/pos-action-delegate-select-validator/pos-action-delegate-select-validator.tsx

```diff
@@ -0,0 +1,245 @@
+    accountIndex: number;
+    blockchain: Blockchain;
+    token: ITokenState;
+    delegationType: DelegationType;
+    validators: IValidator[];
+    title: string;
+}
+
+interface IState {
+    headerSteps: IHeaderStep[];
+    nrValidators: number;
+    validatorsList: IValidator[];
+}
+
+export const navigationOptions = ({ navigation }: any) => ({
+    title: translate(navigation.state.params.title || 'App.labels.send')
+});
+export class PosActionSelectValidatorComponent extends React.Component<
+    INavigationProps<INavigationParams> &
+        IReduxProps &
+        IThemeProps<ReturnType<typeof stylesProvider>>,
+    IState
+> {
+    public static navigationOptions = navigationOptions;
+
+    constructor(
+        props: INavigationProps<INavigationParams> &
+            IReduxProps &
+            IThemeProps<ReturnType<typeof stylesProvider>>
+    ) {
+        super(props);
+
+        const blockchainInstance = getBlockchain(props.account.blockchain);
+
+        const stepList = [];
+        blockchainInstance.config.ui.token.sendStepLabels.map((step, index) => {
+            stepList.push({
+                step: index,
+                title: translate(step),
+                active: index === 0 ? true : false
+            });
+        });
+
+        this.state = {
+            nrValidators: 1,
+            headerSteps: stepList,
+            validatorsList: props.validators
+        };
+    }
+
+    @bind
+    public onSelect(validator: IValidator) {
+        let selected = true;
+        if (validator.actionTypeSelected) {
+            selected = !validator.actionTypeSelected;
+        }
+
+        const validators = this.state.validatorsList;
+        Object.values(validators).map(object => {
+            if (validator.id === object.id) object.actionTypeSelected = selected;
+        });
+        this.setState({ validatorsList: validators });
+    }
+
+    private renderValidatorList() {
+        const { styles } = this.props;
+        const blockchainInstance = getBlockchain(this.props.blockchain);
+        return [
+            <View key={'increase-list'} style={styles.actionContainer}>
+                <TouchableOpacity
+                    style={styles.actionIconContainer}
+                    onPress={() => {
+                        if (this.state.nrValidators > 1) {
+                            const nrValidatorsNew = this.state.nrValidators - 1;
+                            blockchainInstance
+                                .getStats(this.props.chainId)
+                                .getValidatorList(CardActionType.NAVIGATE, nrValidatorsNew)
+                                .then(validators => {
+                                    this.setState({
+                                        nrValidators: nrValidatorsNew,
+                                        validatorsList: validators
+                                    });
+                                })
+                                .catch();
```

Comment:
Just a reminder, if you consider this error from catch to be useful, we can catch this error to sentry.